docs/configuring-metabase/environment-variables.md

```diff
@@ -77,7 +77,7 @@ Only available in [some plans](https://www.metabase.com/pricing)<br>
 Type: string<br>
 Default: `"{}"`
 
-JSON object of primary colors used in charts and throughout Metabase.
+JSON object of primary colors used in charts and throughout Metabase. E.g.: {"brand": "#ff003b", "filter": "#FF003B", "summarize": "#FF003B"} if you want to change the user interface colors, or {"accent0": "#FF0005", "accent1": "#E6C367", "accent2": "#B9E68A", "accent3": "#8AE69F", "accent4": "#8AE6E4", "accent5": "#8AA2E6", "accent6": "#B68AE6", "accent7": "#E68AD0"} if you want to change the chart colors.
```

Comment:
```suggestion
JSON object of primary colors used in charts and throughout Metabase. E.g.: `{"brand": "#ff003b", "filter": "#FF003B", "summarize": "#FF003B"}` if you want to change the user interface colors, or `{"accent0": "#FF0005", "accent1": "#E6C367", "accent2": "#B9E68A", "accent3": "#8AE69F", "accent4": "#8AE6E4", "accent5": "#8AA2E6", "accent6": "#B68AE6", "accent7": "#E68AD0"}` if you want to change the chart colors.
```

Was this supposed to be formatted as code?
Even better than this sufeestion would be to use code blocks, IMO.

Reply:
Agree, thanks!